app/src/main/res/layout/admin_auth_activity.xml

```diff
@@ -14,7 +14,8 @@
     android:id="@+id/admin_auth_container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/addProfileBackground">
+    android:background="@color/addProfileBackground"
+    android:paddingBottom="92dp">
```

Comment:
@bladebunny Please remove `paddingBottom`

paddingBottom was part of only `landscape-layout` and not `portrait layout`.

Reply:
Done